BLSQ/iaso
# IA-2053 : workflow version changes/follow-up order

On a workflow details page, follow-ups are not sorted by order and changes are nor not sortable.

Related JIRA tickets : IA-2053

## Self proofreading checklist


## Changes

- change workflow detail api to order follow-ups by order field
- moved changes result from detail cal to a dedicated one, allowing ordering of results
- plug new api features to front-end

## How to test

- Edit a existing non draft workflow version with multiple changes and follow-ups 
- follow-ups should be displayed ordered on order field
- you should be able to change order of changes by clicking in a head cell

## Print screen / video


https://user-images.githubusercontent.com/12494624/232487806-c0baefe8-f81a-4607-8482-96c4e2cb03d2.mov
